apps/src/util/experiments.js

```diff
@@ -45,6 +45,8 @@ experiments.GENDER_FEATURE_ENABLED = 'gender';
 // Experiment for enabling the CPA lockout
 experiments.CPA_EXPERIENCE = 'cpa_experience';
 experiments.AI_RUBRICS = 'ai-rubrics';
+// Experiment for enabling the Google Blockly modal function editor
+experiments.MODAL_FUNCTION_EDITOR = 'modalFunctionEditor';
```

Comment:
Nice! So to see function editor on google Blockly Sprite lab, we can use: enableExperiments=modalFunctionEditor&blocklyVersion=google
Wasn't sure, but it works!

Reply:
yep! And once you enable the modal function editor experiment it stays enabled until you disable it, so you don't always need to include that flag (just the first time).

Reply:
Interesting that it's not the case with 'blocklyVersion=google'.